Docs/sphinx_tutorials/source/index.rst

```diff
@@ -35,6 +35,8 @@ The amrex/Tutorials directory is broken into the following categories:
    MUI_Tutorial
    Particles_Tutorial
    SDC_Tutorial
+   SENSEI_Tutorial
```

Comment:
I believe SENSEI was removed

Reply:
Removed from the Tutorials only or from AMReX altogether?

Reply:
I think that line changed because of the original history, in the final PR the SENSEI_Tutorial line isn't there